extensions-support/azure-core-http-client-vertx/deployment/pom.xml

```diff
@@ -101,11 +101,6 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>io.projectreactor</groupId>
-            <artifactId>reactor-test</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
This does not seem to be required. Is it really a leftover from some experiments, @jamesnetherton? 
Otherwise I'd have to find a way how to manage the version.

Reply:
Yes, probably a leftover. The test suite seems to run ok without it.